.github/workflows/macos-checks.yml

```diff
@@ -1,103 +0,0 @@
-name: macOS checks
```

Comment:
IMHO we should keep this workflow, we support macos and we should fix and enable this workflow... at some point in time...

Reply:
We decided to remove worfklows that are not running/leveraged right now. What I can do is tag this commit so we will be able to come back to it. WDYT?

Reply:
alright